bodhi/server/models.py

```diff
@@ -1309,6 +1329,39 @@ def signed(self):
             return True
         return all([build.signed for build in self.builds])
 
+    @property
+    def ci_passed(self):
+        """ Returns a boolean representing if all the builds associated with
+        this update are ignored or have passed their CI testing.
+
+        Returns False if at least one of the tests of the builds associated
+        with this update are queued, running or failed.
+        """
+        if self.ci_status in (
+                CiStatus.ignored.description, CiStatus.passed.description):
+            return True
+        return False
+
+    @property
+    def ci_status(self):
+        """ Returns a label representing if all the builds associated with
+        this update are ignored or have passed their CI testing.
+
+        Returns one of: ignored, passed, failed, queued, running, None
+        """
+
+        ci_status = set([build.ci_status for build in self.builds])
+        if ci_status == set([CiStatus.ignored]):
+            return CiStatus.ignored.description
```

Comment:
When looking at the output of btest, I didn't see this line to be missing, how do you check it?